pyttsx3/drivers/_espeak.py

```diff
@@ -528,16 +530,31 @@ def ListVoices(voice_spec=None):
 if __name__ == "__main__":
 
     def synth_cb(wav, numsample, events):
-        print(numsample, end="")
+        print(f"Callback received: numsample={numsample}")
```

Comment:
This f-string shortcut does the same thing.
```suggestion
        print(f"Callback received: {numsample=}")
```